frontend/src/components/CustomerReviewPage/CustomerReviewPage.js

```diff
@@ -145,34 +145,34 @@ const Questions = ({ questions, answerSheet, onAnswerUpdate }) => {
   )
 }
 
-class CustomerReviewPage extends React.Component {
-  async componentDidMount() {
-    const id = this.props.match.params.id
+const CustomerReviewPage = (props) => {
+  useEffect(async () => {
```

Comment:
Using async directly in the useEffect function isn't allowed, the resulting console.error provides a workaround:

```
It looks like you wrote useEffect(async () => ...) or returned a Promise. Instead, write the async function inside your effect and call it immediately:

useEffect(() => {
  async function fetchData() {
    // You can await here
    const response = await MyAPI.getData(someId);
    // ...
  }
  fetchData();
}, [someId]); // Or [] if effect doesn't need props or state

```